### PR TITLE
🔎 Search bar yay! 🔎

### PR DESCRIPTION
**Pull request by mudziok**
This code is bugged - if you hover datalist inside the sidebar it closes the sidebar. Im pretty sure this is the issue of ActiveZone as the datalist is portalled somewhere else in the dom.

After like an hour of trying I concede defeat as the syntax of the activezone stuff is so cryptic that I simply cannot figure this out XD

### CODE REVIEW

**Comment by mudziok** [comment]
Ofc we'll have to change this once the algolia like search is in place

**Review by AdrianKokot — APPROVED** [review]
As we now have something useful in the searchbar, I think the menu button should be always visible, because user might want to search something before entering any university's page.

Create issue with the bug you mentioned, I'll try to fix it.

File: frontend/modules/shared/ui/src/lib/layout/search/search.component.html
Comment (AdrianKokot) [inline frontend/modules/shared/ui/src/lib/layout/search/search.component.html:1]: To resolve #69 you can add two lines:

```html
<tui-input
   #input <!-- This -->
>
<a
  (click)="input.open = false" <!-- And this -->

```

Comment (AdrianKokot) [inline frontend/modules/shared/ui/src/lib/layout/search/search.component.html:1]: And to resolve #70 you can move template binding from `*tuiLet` to `*ngFor`:

```html
<tui-input
  *tuiLet="results$ | async as results" <!-- Remove this line --->

<a
  *ngFor="let page of results$ | async" <!-- And use async pipe here --->
```

**Review by AdrianKokot — CHANGES_REQUESTED** [review]

File: frontend/modules/shared/ui/src/lib/layout/search/search.component.html
Comment (mudziok) [reply frontend/modules/shared/ui/src/lib/layout/search/search.component.html:1]: Thank you

Comment (mudziok) [reply frontend/modules/shared/ui/src/lib/layout/search/search.component.html:1]: 🤯

**Review by AdrianKokot — APPROVED** [review]

**Pull request merged**